android/uhabits-android/build.gradle

```diff
@@ -87,7 +87,7 @@ dependencies {
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation "com.github.paolorotolo:appintro:3.4.0"
     implementation "com.google.dagger:dagger:$DAGGER_VERSION"
-    implementation "com.jakewharton:butterknife:8.6.1-SNAPSHOT"
+    implementation "com.jakewharton:butterknife:10.0.0"
```

Comment:
Please revert these changes, they are not related to the functionality implemented in this PR.